### PR TITLE
Add pandas example

### PR DESCRIPTION
**Pull request by justinzw**
This is a example that takes a response to_data() method then outputs it to two files.
One is a json file with the raw dict in a json format and then another is a table that was built using pandas

### CODE REVIEW

**Review by thakkerurvish — APPROVED** [review]
Overall LGTM! Cool integration with pandas:)

Just some nits, to address!

File: examples/leveraging_to_data_method/pandas.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+
+import webql
+import json
+import pandas as pd
+import logging
+from webql.sync_api.web import PlaywrightWebDriver
+
+
+logging.basicConfig(level=logging.DEBUG)
```
[the remark this answers is not in the document]
Comment (justinzw) [reply examples/leveraging_to_data_method/pandas.py:12]: For the logging? It's used to send debug logs.

File: examples/leveraging_to_data_method/pandas.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        colors
+    }
+    
+}
+"""
+response = session.query(QUERY)
+data = response.shoes.to_data()
+
+#Let's dump the data out to a json file for us to use.
+
```
[the remark this answers is not in the document]
Comment (justinzw) [reply examples/leveraging_to_data_method/pandas.py:34]: removed

File: examples/leveraging_to_data_method/pandas.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (thakkerurvish) [reply examples/leveraging_to_data_method/pandas.py:12]: I meant, there no logs as of now in the script so its unused. This line just configures the logger and not actually send any logs.

**Pull request merged**